internal/template-validator/validation/eval.go

```diff
@@ -39,7 +39,7 @@ var (
 	ErrUnsatisfiedRule      = errors.New("rule is not satisfied")
 )
 
-type Report struct {
+type ValidationReport struct {
```

Comment:
This change is not related to tests.

Reply:
`Report` is already declared by `Ginkgo v2`. Apologies I did mean to leave a note about this in the commit, I'll add one now:

```
$ git checkout origin/master internal/template-validator/validation/eval.go
Updated 1 path from f923d2dd
[08:55:03]lyarwood@hobbiton.yarwood.me.uk:~/redhat/devel/src/k8s/kubevirt/ssp-operator ginkgo-v2# make unittest 
test -s /home/lyarwood/redhat/devel/src/k8s/kubevirt/ssp-operator/bin/controller-gen || GOBIN=/home/lyarwood/redhat/devel/src/k8s/kubevirt/ssp-operator/bin go install sigs.k8s.io/controller-tools/cmd/controller-gen@v0.10.0
/home/lyarwood/redhat/devel/src/k8s/kubevirt/ssp-operator/bin/controller-gen object:headerFile="hack/boilerplate.go.txt" "paths={./controllers/..., ./internal/..., ./hack/..., ./webhooks/...}"
cd api && /home/lyarwood/redhat/devel/src/k8s/kubevirt/ssp-operator/bin/controller-gen object:headerFile="../hack/boilerplate.go.txt" "paths=./..."
go fmt ./...
go vet ./...
# kubevirt.io/ssp-operator/internal/template-validator/validation
vet: internal/template-validator/validation/eval.go:42:6: Report already declared through dot-import of package ginkgo ("github.com/onsi/ginkgo/v2")
make: *** [Makefile:155: vet] Error 2
```